### PR TITLE
Add GEN-7 pokemon zh-hans names.

### PR DESCRIPTION
**Pull request by Dafrok**

### CODE REVIEW

**Comment by sindresorhus** [comment]
@CalvertYang Can you review?

File: data/zh-hans.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [
-	"妙蛙种子",
+ 	"妙蛙种子",
```
Comment (sindresorhus) [inline data/zh-hans.json]: You changed some whitespace here.

**Comment by CalvertYang** [comment]
@sindresorhus Translations looks good

File: data/zh-hans.json [hunk 2]
```diff
@@ -719,5 +719,86 @@
...
+	"沙丘娃‎",
+	"噬沙堡爷",
+	"拳海参‎",
+	"属性：空",
+	"银伴战兽",
+	"小陨星",
+	"树枕尾熊",
+	"爆焰龟兽",
+	"托戈德玛尔",
+	"谜拟Ｑ",
```
Comment (CalvertYang) [inline data/zh-hans.json]: @sindresorhus Half-width or full-width of the letter which is better?

Comment (sindresorhus) [inline data/zh-hans.json]: I don't know to be honest. Whatever the most common for Chinese on the web is.

Comment (CalvertYang) [inline data/zh-hans.json]: I just found that Nintendo Ltd. use full-width alphabet on the web.

**Comment by sindresorhus** [comment]
ping @Dafrok => https://github.com/sindresorhus/pokemon/pull/23#discussion_r89535300

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Comment by Dafrok** [comment]
Need some help? Sorry for lacking of response. There're too much notifications I have to read.

**Pull request closed without merging**